pysat/instruments/pysat_testing.py

```diff
@@ -101,9 +92,8 @@ def preprocess(self):
 
 
 def load(fnames, tag=None, inst_id=None, sim_multi_file_right=False,
-         sim_multi_file_left=False, root_date=None, file_date_range=None,
-         malformed_index=False, mangle_file_dates=False, num_samples=None,
-         multi_file_day=False):
+         sim_multi_file_left=False, root_date=None, malformed_index=False,
+         num_samples=None, multi_file_day=False):
```

Comment:
the `multi_file_day` keyword doesn't go to anything that I can see. I wasn't directly in on the recent-is `multi_file_day` change, perhaps this is a holdover?

Reply:
That sounds highly possible.